site/docs/reference/working-logs-stats.md

```diff
@@ -6,65 +6,50 @@ sidebar_position: 3
 This method allows more flexibility and is ideal for debugging.
 
-You can retrieve logs for any task that is part of a catalog that is currently deployed.
-
-#### Printing logs for a specific task
-
-You can print logs for a given deployed task using the flag `--task` followed by the task name.
+You can retrieve logs and stats for any published Flow task. For example:
 
 ```console
 flowctl logs --task acmeCo/anvils/capture-one
-```
-
-#### Printing all logs for a prefix
 
-You can print all logs for currently deployed catalogs of a given prefix using the flag `--tenant`.
-
-```console
-flowctl logs --tenant acmeCo
+flowctl stats --task acmeCo/anvils/capture-one --uncommitted
 ```
 
-This is the same as printing the entire contents of the collection `ops/acmeCo/logs`.
+:::info Beta
+The `--uncommitted` flag is currently required for `flowctl stats`. This means that all statistics are read, regardless of whether they are about a successfully committed [transaction](../concepts/advanced/shards.md#transactions), or a transaction that was rolled back or uncommitted.
+In the future, committed reads will be the default.
+:::
 
-#### Printing logs by task type
+#### Printing logs or stats since a specific time
 
-Within a given prefix, you can print logs for all deployed tasks of a given type using the flag `--task-type` followed by one of `capture`, `derivation`, or `materialization`.
+To limit output, you can retrieve logs are stats starting at a specific time in the past. For example:
 
-```console
-flowctl logs --tenant acmeCo --task-type capture
 ```
+flowctl stats --task acmeCo/anvils/materialization-one --since 1h
+```
+
+...will retrieve stats from approximately the last hour. The actual start time will always be at the previous [fragment](../concepts/advanced/journals.md#fragment-files) boundary, so it can be significantly before the requested time period.
 
-### Accessing logs by materialization
+Additional options for `flowctl logs` and `flowctl stats` can be accessed through command-line help.
 
-You can materialize your `logs` collection to an external system.
-This is typically the preferred method if you’d like to continuously work with or monitor logs.
-It's easiest to materialize the whole collection, but you can use a [partition selector](../../concepts/materialization/#partition-selectors) to only materialize specific tasks, as the `logs` collection is partitioned on tasks.
+### Accessing logs or stats by materialization
+
+You can materialize your `logs` or `stats` collections to an external system.
+This is typically the preferred method if you’d like to continuously work with or monitor logs or statistics.
+It's easiest to materialize the whole collection, but you can use a [partition selector](../../concepts/materialization/#partition-selectors) to only materialize specific tasks, as the `logs` and `stats` collections are partitioned on tasks.
```

Comment:
> It's easiest to materialize the whole collection

This seems to contradict the advice below about excluding the logs/stats of the materialization itself, since doing so requires a partition selector. I think it'd probably be best to just say that you can materialize the logs/stats from all tasks, or just a subset of them.